README.md

```diff
@@ -46,7 +46,7 @@ export default defineConfig({
 
 This plugin is necessary to run portable stories in Vitest, as it provides the necessary Next.js features to ensure that your components are tested in an environment that closely mirrors production.
 
-#### Experimental @storybook/experimental-vitest-plugin
+### Experimental @storybook/experimental-vitest-plugin
```

Comment:
@kylegach I was wondering if you could switch the order of this part with the configuration options below and adjust the headings accordingly so that users have upfront what they need to do if they want to extend their configuration based on their environment.